configs/server/forseti_conf_server.yaml

```diff
@@ -0,0 +1,206 @@
+global:
```

Comment:
@blueandgold Here is a yaml file that will minimize inventories done outside of gsuite

Reply:
Okay, got it... but we don't want to check this in, so can you please remove from this PR.